camp/camp-brooklyn/src/main/java/org/apache/brooklyn/camp/brooklyn/spi/dsl/methods/DslComponent.java

```diff
@@ -410,6 +424,7 @@ public String toString() {
         }
     }
 
+    @DslAccessible
```

Comment:
How is it that `attributeWhenReady()` and `config()` are able to be `@DslAccessible` when a few commits ago they did not return a value that implemented `DslCallable`?